app/src/main/java/com/samco/trackandgraph/graphstatinput/GraphStatInputView.kt

```diff
@@ -179,73 +178,68 @@ private fun GraphStatInputViewForm(
     viewModelStoreOwner: ViewModelStoreOwner,
     viewModel: GraphStatInputViewModel,
     graphStatId: Long,
-    scrollState: ScrollState = rememberScrollState()
-) = Column(
-    modifier = modifier
-        .padding(
-            top = dimensionResource(id = R.dimen.card_padding),
-            start = dimensionResource(id = R.dimen.card_padding),
-            end = dimensionResource(id = R.dimen.card_padding)
+    scrollState: ScrollState = rememberScrollState(),
+    isInErrorState: Boolean,
+    isInUpdateMode: Boolean
+) = FormSurface (scrollState = scrollState) {
+    Column {
+        FormLabel(text = stringResource(id = R.string.graph_or_stat_name))
+        FormTextInput(
+            textFieldValue = viewModel.graphName,
+            onValueChange = { viewModel.setGraphStatName(it) }
         )
-        .verticalScroll(state = scrollState)
-) {
 
-    FullWidthTextField(
```

Comment:
It seems lik FullWidthTextField is kind of being replaced with FormTextInput. It's probably worth looking at the other places it's used and seeing if we can get away with swapping it entirely and removing it? Doesn't have to be in this PR though.